src/Symfony/Component/Process/InputStream.php

```diff
@@ -20,12 +20,15 @@
  */
 class InputStream implements \IteratorAggregate
 {
+    /** @var null|callable */
     private $onEmpty = null;
-    private $input = array();
+    private $input = [];
```

Comment:
you should revert these changes, we always use `array()` in the code base for now

Reply:
Oh sorry. Wasn't supposed to be there. My autoformat just changed it. Will revert it